src/openzaak/components/documenten/management/commands/delete_empty_eioc.py

```diff
@@ -0,0 +1,60 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2024 Dimpact
+from typing import List
+
+from django.conf import settings
+from django.core.management import BaseCommand
+from django.db import transaction
+from django.utils.translation import gettext_lazy as _
+
+from openzaak.components.besluiten.models import BesluitInformatieObject
+from openzaak.components.documenten.models import EnkelvoudigInformatieObjectCanonical
+from openzaak.components.zaken.models import ZaakInformatieObject
+
+
+class Command(BaseCommand):
+    help = "Check for any empty document identities (no related document)."
+
+    def handle(self, *args, **options):
+        if settings.CMIS_ENABLED:
+            self.stdout.write(_("This command does not run with CMIS enabled."))
+            return
+
+        msg = _("Checking {count} records ...").format(
+            count=EnkelvoudigInformatieObjectCanonical.objects.count()
+        )
+        self.stdout.write(msg)
+
+        duplicates = EnkelvoudigInformatieObjectCanonical.objects.filter(
+            enkelvoudiginformatieobject__isnull=True
```

Comment:
Do we know where these canonicals without version come from? Are they created by admins?